clients/src/main/java/org/apache/kafka/common/telemetry/internals/ClientTelemetryReporter.java

```diff
@@ -623,14 +622,7 @@ public void initiateClose(long timeoutMs) {
                     return;
                 }
 
-                try {
-                    log.info("About to wait {} ms. for terminal telemetry push to be submitted", timeoutMs);
```

Comment:
You don't actually make use of the `timeoutMs` argument any more (apart from in a debug log line). Probably ought to remove it.

Reply:
Done.